backend/controllers/org.controller.ts

```diff
@@ -0,0 +1,47 @@
+import { OrganizationCreateSchema } from "../../shared/zodSchemas";
```

Comment:
might have to coordinate changes with this with Noman, but I'm fine with it for now.